docs/articles/expensify-classic/expensify-billing/Billing-Overview.md

```diff
@@ -26,15 +26,18 @@ If at least 50% of your approved USD spend in a given month is on your company
 Additionally, every month, you receive 1% cash back on all Expensify Card purchases, and 2% if the spend across your Expensify Cards is $250k or more (_applies to USD purchases only_). Any cash back from the Expensify Card is first applied to your Expensify bill, further reducing your price per member. Any leftover cash back is deposited directly into your connected bank account.
 ## Savings calculator
 To see how much money you can save (and even earn!) by using the Expensify Card, check out our [savings calculator](https://use.expensify.com/price-savings-calculator). Just enter a few details and see how much you’ll save!
+
 {% include faq-begin.md %}
+
 ## What if we put less than 50% of our total spend on the Expensify Card?
-If you put less than 50% of your total USD spend on your Expensify Card, your bill gets discounted on a sliding scale based on the percentage of use. So if you don't use the Expensify Card at all, you'll be charged the full rate for each member based on your plan and subscription.
-Example:
+If you spend less than 50% of your total USD on your Expensify Card, your bill gets discounted on a sliding scale based on the percentage of use. So, if you don't use the Expensify Card at all, you'll be charged the full rate for each member based on your plan and subscription.
```

Comment:
I don't think this sentence reads very well or makes much sense.

"If you spend less than 50% of your total USD on your Expensify Card". Particularly "of your total USD"

How about this?
"If less than 50% of your total USD spend is on the Expensify Card,"

Reply:
Sure! ya, i just did a super HL overview of this and updated anything egregious. but i like that update, neil!